src/v/coproc/reconciliation_backend.cc

```diff
@@ -109,8 +109,80 @@ ss::future<> reconciliation_backend::fetch_and_reconcile() {
     }
 }
 
-ss::future<std::error_code> reconciliation_backend::process_update(update_t) {
+ss::future<std::error_code>
+reconciliation_backend::process_update(update_t delta) {
+    using op_t = update_t::op_type;
+    model::revision_id rev(delta.offset());
+    switch (delta.type) {
+    case op_t::add_non_replicable:
+        if (!cluster::has_local_replicas(
+              _self, delta.new_assignment.replicas)) {
+            return ss::make_ready_future<std::error_code>(errc::success);
+        }
+        return create_non_replicable_partition(delta.ntp, rev);
+    case op_t::del_non_replicable:
+        return delete_non_replicable_partition(delta.ntp, rev).then([] {
+            return std::error_code(errc::success);
+        });
+    case op_t::add:
```

Comment:
are we going to support moving of `replicated_topics` ?

Reply:
Yes but this will occur by listening for moves of parent topics. This way the move is transparent and implicit. If you move a parent topic to a materialized one, the materialized topics just follow.